cmd/flipt/doc.go

```diff
@@ -23,7 +23,13 @@ func newDocCommand() *cobra.Command {
 				return fmt.Errorf("failed to create docs directory: %w", err)
 			}
 
-			return doc.GenMarkdownTree(cmd.Root(), "./tmp/docs")
+			if err := doc.GenMarkdownTree(cmd.Root(), "./tmp/docs"); err != nil {
```

Comment:
It looks like docs are always generated into ` "./tmp/docs"`. It isn't very clear why we need `[path]` param in this command.

Reply:
good call! fixed

```
workspace/flipt - [minor-cmd-updates●] » ./bin/flipt doc /tmp
Documentation generated in: /tmp
workspace/flipt - [minor-cmd-updates●] » ls /tmp

flipt.md                        flipt_bundle_pull.md            flipt_config_init.md            flipt_migrate.md
flipt_bundle.md                 flipt_bundle_push.md            flipt_evaluate.md               flipt_validate.md
```